packages/react-components/src/organisms/GroupCard.tsx

```diff
@@ -2,38 +2,56 @@ import { css } from '@emotion/react';
 import { GroupResponse } from '@asap-hub/model';
 import { network } from '@asap-hub/routing';
 
-import { Card, Paragraph, Anchor } from '../atoms';
+import { Card, Paragraph, Anchor, StateTag } from '../atoms';
 import { LinkHeadline, TagList } from '../molecules';
 import { teamIcon } from '../icons';
-import { perRem } from '../pixels';
+import { perRem, tabletScreen } from '../pixels';
 
 const iconStyles = css({
   display: 'inline-grid',
   verticalAlign: 'middle',
   paddingRight: `${15 / perRem}em`,
 });
 
+const titleStyle = css({
+  display: 'flex',
+  flexFlow: 'row',
+  gap: `${16 / perRem}em`,
+  alignItems: 'center',
+  marginBottom: `${-12 / perRem}em`,
+  [`@media (max-width: ${tabletScreen.width - 1}px)`]: {
```

Comment:
You should try and develop mobile first. This is like putting in the mobile view changes afterwards. gap: 0 i think is invalid I think you wanted unset.